BlazarUI/app/scripts/components/shared/PageHeader.jsx

```diff
@@ -1,6 +1,19 @@
 import React, {Component, PropTypes} from 'react';
+import $ from 'jquery';
```

Comment:
i'd like as much as we can to avoid jquery and instead use react where we can, e.g. -> https://facebook.github.io/react/docs/animation.html